code/modules/uplink/uplink_item.dm

```diff
@@ -1208,6 +1208,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 20
 	restricted_roles = list("Chaplain")
 	surplus = 5 //Very low chance to get it in a surplus crate even without being the chaplain
+/datum/uplink_item/role_restricted/ancient_jumpsuit
+	name = "Ancient Jumpsuit"
+	desc = "A jumpsuit recovered from a station overcome by the grey tide. Yeah."
```

Comment:
Please give it an actual description so people know its a joke item and dont waste all their points and get mad